setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='coffeehouse',
-    version='2.1.0',
+    version='3.0.0',
```

Comment:
There is no need to push the package version to 3.0.0 for no reason. This pull request is introducing changes from 1.* and pushing it to 3.*

What happens when something new is added? The version is pushed to 4.0.0 because the changes made in 3.0.0 is incompatible or inconsistent with the current structure?